pkg/sql/sem/builtins/builtins.go

```diff
@@ -6535,7 +6535,7 @@ table's zone configuration this will return NULL.`,
 			Category: categorySystemInfo,
 		},
 		tree.Overload{
-			Types:      tree.ArgTypes{{"session", types.Bytes}},
+			Types:      tree.ArgTypes{},
```

Comment:
cc @cockroachdb/sql-observability is there any frontend page that calls this function? if so, we'd have  to make sure that this change  is OK to make.